README.md

```diff
@@ -31,10 +31,14 @@ We welcome you to try it and look forward to feedback on the library! However, p
 
 ## Installation
 
-[OpenDP is on PyPI](https://pypi.org/project/opendp/) and can be installed with `pip` (the [package installer for Python](https://pypi.org/project/pip/)):
+Install OpenDP for Python with `pip` (the [package installer for Python](https://pypi.org/project/pip/)):
 
     $ pip install opendp
 
+Install OpenDP for R from an R session:
+
+    install.packages("opendp", repos = "https://opendp.r-universe.dev")
```

Comment:
I've changed the branch name now, but-- I've configured R-universe to watch for changes on `r-universe` and `r-universe-dev`. When R-universe sees an update to either of those branches, it will pre-compile binaries for those platforms and update the package hosted on https://opendp.r-universe.dev/, or https://opendp-dev.r-universe.dev, respectively.